agent/acl.go

```diff
@@ -41,6 +41,36 @@ func (a *Agent) resolveTokenAndDefaultMeta(id string, entMeta *structs.Enterpris
 	return a.delegate.ResolveTokenAndDefaultMeta(id, entMeta, authzContext)
 }
 
+// resolveTokenAndDefaultMeta is used to resolve an ACLToken's secretID to a structs.ACLIdentity
```

Comment:
```suggestion
// resolveIdentityFromToken is used to resolve an ACLToken's secretID to a structs.ACLIdentity
```

Reply:
Oh yuuuup! Thank you for catching this 🙏 - I think it slipped past my linter because the function isn't exported